### PR TITLE
Slingshot Scavenger Hunt

### PR DESCRIPTION
**Pull request by jimpick**
A collaborative information collection game that uses Filecoin!

### CODE REVIEW

**Comment by dineshshenoy** [comment]
Thanks for submitting an application.  Our team will validate for completeness and then assign reviewers from the competitions review panel.

File: participants/slingshot-scavenger-hunt.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+We may ask our users to register using GitHub accounts as we want to restrict
+access to the Lotus nodes we provide to authorized users. Apart from a small
+amount of data for things such as leaderboards, most data collected will be
+woven into a master IPLD-based knowledge graph. We will focus on public domain
+or creative commons licensed content as much as possible.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+Probably about 10 replicas of the Kiwix dump for redundancy. Players will find and
```
Comment (dineshshenoy) [inline participants/slingshot-scavenger-hunt.md:50]: The rules will be updated to limit data to 3 replicas.  Can you change this parameter?

**Comment by dineshshenoy** [comment]
This is a really cool idea.  If you can change the replica parameter - we can move forward to approve.

File: participants/slingshot-scavenger-hunt.md (hunk 1 above)
Comment (jimpick) [reply participants/slingshot-scavenger-hunt.md]: The new rule is 10 replicas. I changed the parameter to 7.

Comment (dineshshenoy) [reply participants/slingshot-scavenger-hunt.md]: You're correct - sorry about the confusion.  Feel free to change back to 10 if preferred

**Review by dineshshenoy — APPROVED** [review]

**Comment by dineshshenoy** [comment]
Congratulations. Your application has been approved and merged as an official participant for the Slingshot competition!
The following resources may be helpful as you begin:

[Competition Home](https://slingshot.filecoin.io/)
[Competition Rules](https://slingshot.filecoin.io/rules)
[FAQs](https://slingshot.filecoin.io/faq)
[Events calendar](https://slingshot.filecoin.io/events/) (Office Hours/Master Classes)
[Filecoin Docs](https://docs.filecoin.io) 
[HackFS videos](https://hack.ethglobal.co/showcase/myel-receOUiPSd63148qZ)
[Textile Powergate](https://docs.textile.io/powergate/)

Feel free to reach out in the slack community channels:
#fil-testnet-announce – For major announcements about the testnet and major software releases
#slingshot-announcements – For major announcements about the Slingshot program
#slingshot – For debugging help and community conversations throughout the course of the competition

**Pull request merged**